tool_commands/commands/Step 3 - Modify schematisation/guess_indicators.py

```diff
@@ -1,10 +1,12 @@
 # (c) Nelen & Schuurmans, see LICENSE.rst.
 
-from ThreeDiToolbox.tool_commands.base.custom_command import CustomCommandBase
-from ThreeDiToolbox.utils.guess_indicators import Guesser
+from ThreeDiToolbox.tool_commands.custom_command_base import CustomCommandBase
+from ThreeDiToolbox.tool_commands.guess_indicators.guess_indicator_view import (
```

Comment:
Even kijken of je op deze (en een paar andere plekken) weer de `.... import guess_indicator_view` truc toe kunt passen om de lange imports wat in te perken.